oap-stdlib/src/test/resources/oap/application/KernelProfileTest/module.yaml

```diff
@@ -5,7 +5,7 @@ services:
     implementation: "oap.application.KernelProfileTest$TestProfile1"
 
   profile2:
```

Comment:
Is it better to rename `profile1` and `profile2` to `service1` and `service2`?

Reply:
Agree. I will rename it in separate PR as it's better to rename it everywhere